solutions/usaco-1059.mdx

```diff
@@ -8,6 +8,46 @@ author: Sathvik Chundru
 [Official Analysis](http://www.usaco.org/current/data/sol_prob1_bronze_dec20.html)
 
 <LanguageSection>
+
+<CPPSection>
+
+```cpp
```

Comment:
i think the official analysis is very clear, so this is unnecessary...

Reply:
should i just delete it then 
the whole cpp solution

Reply:
ah rip here it goes then